README.md

```diff
@@ -1,2 +1,23 @@
-# librosa_gallery
-A collection of example notebooks demonstrating librosa functionality
+# Librosa Gallery
+
+A collection of code examples demonstrating some of librosa's functionality.
+
+## Examples
+
+
+## Notebooks
+
```

Comment:
I keep the notebooks in this repo for reference and ease of development, but I think it's better to refer folks to the sphinx-gallery build https://librosa.github.io/librosa_gallery/